flow.js

```diff
@@ -429,6 +429,7 @@ declare module 'brewskey.js-api' {
   */
```

Comment:
I would rather that you only export the things that are necessary instead of including all the class declarations in the export module.